easybuild/easyconfigs/r/reframe/reframe-3.4.2.eb

```diff
@@ -0,0 +1,48 @@
+# @author: gppezzi, vkarak, manitart
+
+easyblock = 'Tarball'
+
+name = 'reframe'
+version = '3.4.2'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = 'A regression framework for writing portable tests for HPC systems.'
+
+toolchain = SYSTEM
+
+sources = [
+    {
+        'filename': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/eth-cscs/reframe/archive/']
+    },
+    {
+        'filename': '/apps/common/easybuild/sources/%(nameletterlower)s/'
+                    '%(name)s/reframe-tests-%(version)s.tar.gz'
+    }
+]
+
+keepsymlinks = True
+postinstallcmds = [
+    'cp -r %(builddir)s/reframe-tests-%(version)s/checks %(installdir)s/cscs-checks/private',
+    './bootstrap.sh +docs +pygelf && '
+    'cp -r external %(installdir)s/external && '
+    'mkdir -p %(installdir)s/share/man/man1 %(installdir)s/share/man/man8 && '
+    'cp docs/man/man1/reframe.1 %(installdir)s/share/man/man1 && '
+    'cp docs/man/man8/reframe.settings.8 %(installdir)s/share/man/man8'
```

Comment:
Do we need to copy now also the shell competions? Check here: https://reframe-hpc.readthedocs.io/en/stable/started.html#enabling-auto-completion

Reply:
No, they will not work, but they should be there so that users can source them according to the documentation.

Reply:
Ok, I will add the corresponding command and push